lms/djangoapps/certificates/generation_handler.py

```diff
@@ -177,11 +183,6 @@ def _can_generate_allowlist_certificate(user, course_key):
         log.info(f'{user.id} does not have a verified id. Allowlist certificate cannot be generated for {course_key}.')
         return False
 
-    if not _is_on_certificate_allowlist(user, course_key):
```

Comment:
Is this move to simplify the check and speed up rejection if users are not in the list?

Reply:
Yep.